libs/hdf-converters/src/asff-mapper/asff-mapper.ts

```diff
@@ -37,29 +43,22 @@ export enum SpecialCasing {
 function whichSpecialCase(finding: Record<string, unknown>): SpecialCasing {
   const productArn = _.get(finding, 'ProductArn') as string;
   if (
-    productArn.match(
-      /^arn:[^:]+:securityhub:[^:]+:[^:]*:product\/aws\/firewall-manager$/
-    )
-  ) {
-    return SpecialCasing.FirewallManager;
-  } else if (
-    productArn.match(
-      /^arn:[^:]+:securityhub:[^:]+:[^:]*:product\/prowler\/prowler$/
-    )
+    _.get(finding, 'ProductName') === 'Default' &&
+    _.get(finding, 'GeneratorId') === 'cms.Chef Inspec'
   ) {
-    return SpecialCasing.Prowler;
+    return SpecialCasing.CMSInSpec;
   } else if (
     productArn.match(
-      /^arn:[^:]+:securityhub:[^:]+:[^:]*:product\/aws\/securityhub$/
+      /^arn:[^:]+:securityhub:[^:]+:[^:]*:product\/aws\/firewall-manager$/
     )
   ) {
-    return SpecialCasing.SecurityHub;
+    return SpecialCasing.FirewallManager;
   } else if (
     productArn.match(
-      /^arn:[^:]+:securityhub:[^:]+:[^:]*:product\/aquasecurity\/aquasecurity$/
+      /^arn:[^:]+:securityhub:[^:]+:[^:]*:product\/aws\/guardduty$/
```

Comment:
Just looking at consistency - firewall-manager is separated by a dash and guardduty isn't. Maybe that is correct, but just want to make sure it is intentional.

Reply:
It is intentional.  It all depends on how that application registered itself as a product with AWS SecHub.  Guardduty doesn't have a dash, firewall manager does.